app/assets/stylesheets/blacklight/_facets.scss

```diff
@@ -148,6 +148,22 @@ ul.facet_extended_list, .facet_extended_list ul
   @extend .clearfix;
 }
 
+ 
+// Facet modal alphabetical filter
+.alpha-filter {
+  margin-bottom: 18px;
```

Comment:
`18px` seems oddly specific. Are we missing a native bootstrap class that would provide this? Or is there a bootstrap-provided scss variable we should use instead?